spec/article_json/export/apple_news/elements/image_spec.rb

```diff
@@ -0,0 +1,42 @@
+describe ArticleJSON::Export::AppleNews::Elements::Image do
+  subject(:element) { described_class.new(source_element) }
+
+  let(:source_element) do
+    ArticleJSON::Elements::Image.new(
+      source_url: '/foo/bar.jpg',
+      caption: caption,
+      float: float,
+      alt: alt
+    )
+  end
+  let(:float) { nil }
+  let(:caption) { [ArticleJSON::Elements::Text.new(content: 'Foo Bar')] }
+  let(:alt) { 'Text description' }
+
+  let(:expected_json) { {:URL=>"/foo/bar.jpg", :caption=>"Foo Bar", :role=>"image"} }
```

Comment:
How about `{ URL: "/foo/bar.jpg", caption: "Foo Bar", role: "image" }` ?

Reply:
Yep, we should follow the style we use elsewhere (also using single quotes):
```suggestion
      let(:expected_json) { {URL: '/foo/bar.jpg', caption: 'Foo Bar', role: 'image'} }
```

Reply:
Amended